tasks/lib/notice/__tests__/notice.js

```diff
@@ -9,11 +9,11 @@ const NODE_MODULES = resolve(__dirname, '../../../../node_modules');
 const NODE_DIR = resolve(process.execPath, '../..');
 const PACKAGES = [
   {
-    name: '@elastic/httpolyglot',
-    version: '0.1.2-elasticpatch1',
+    name: '@spalger/filesaver',
```

Comment:
Let's switch this to being a controlled package that we know will always exist because we've created it specifically for this purpose.  We can create an npm package under the @elastic org that does nothing except provide the license we want to verify against, and then we can depend on that package in our devDependencies.  We should never need to update this test just because we decided to no longer depend on a given package in our source.